ci/build.bat

```diff
@@ -42,19 +48,19 @@ where.exe neovim-node-host.cmd || goto :error
 
 mkdir .deps
 cd .deps
-cmake -G "MinGW Makefiles" -DCMAKE_BUILD_TYPE=RelWithDebInfo ..\third-party\ || goto :error
-mingw32-make VERBOSE=1 || goto :error
+cmake -G %CMAKE_GENERATOR% -DCMAKE_BUILD_TYPE=RelWithDebInfo ..\third-party\ || goto :error
```

Comment:
should `%CMAKE_GENERATOR%` be in quotes?

Reply:
Right, but the quotes aren't implicit. Quotes are not needed on that line because they are part of the value assigned here:
```batch
set CMAKE_GENERATOR="Visual Studio 15 2017"
```
Batch is a very strange scripting language and it has mostly been made obsolete by PowerShell.

Reply:
`set "CMAKE_GENERATOR=Visual Studio 15 2017"`